README.md

```diff
@@ -4,7 +4,12 @@ Command line tools for [Cloudflare](https://cloudflare.com)
 
 ## Installation
 
-To install, place `cloudflare-cli` anywhere in your PATH. Optionally alias it to something easier, like `cf`.
+To install, place `cloudflare-cli` anywhere in your PATH. Optionally alias it to something easier, like `cfcli`.
+Or just run the following command to install it as `cfcli`.
+
+```bash
+sudo wget -qO cfcli https://git.io/cfcli && sudo chmod +x ./cfcli && sudo install ./cfcli /usr/local/bin/cfcli
```

Comment:
two things:

1. i don't think these first two commands need to be run as `sudo`. and the third command only needs to be if you install it in `/usr/local/bin`, but this could just as easily be in `~/` and avoid the need for sudo
2. i'm a bit uneasy with using `git.io` here, since it's opaque what script will end up being installed. could you change this to be `https://raw.githubusercontent.com/thelowlypeon/cloudflare-cli/master/cloudflare-cli`?